foundation/foundation-mda/src/test/resources/specifications/pipeline-generation.feature

```diff
@@ -47,3 +47,10 @@ Feature: Generating pipeline steps
 		When the profile "data-delivery-pyspark-pipeline" is generated
 		Then a common step parent is created
 		And the common step parent is customizable
+
+	Scenario: PySpark Data Ingest steps in data-flow pipelines can write to a RDMS database
```

Comment:
Should the test be for postgressql? From the code, it looks like there are distinctive differences between RDMS and Postgressql.

Reply:
As per the Peter's comment above, RDBMS as a type is intended to replace the postgres type as postgres is a RDBMS option. I was being cautious and holding out on completely pruning it in case they needed a migration in downstreams, but there are no breaking changes to any impl files so it'd be best to pull out the postgres type.